assets/aml-benchmark/components/src/inference_postprocessor/runner.py

```diff
@@ -0,0 +1,240 @@
+# ---------------------------------------------------------
```

Comment:
This probably does not belong to this repo either.

Reply:
Added to babel repo.